app/src/Pages/Portal/Portal.js

```diff
@@ -4,36 +4,72 @@ import { CSSTransition, TransitionGroup } from "react-transition-group";
 import DashboardPage from "./Dashboard/Dashboard";
 import ContributionsPage from "./Contributions/Contributions";
 import PageHoc from "../../components/PageHoc/PageHoc";
+import Sidebar from "../../components/Sidebar";
+
+/* @jsx jsx */
+import { css, jsx } from '@emotion/core';
+
+const styles = css`
+
+  &:after {
+    content: '';
+    clear: both;
+    display: table;
+  }
+  .sidebar-wrapper {
+    width: 20%;
+    position: sticky;
+    top: 0;
+    left: 0;
+  }
+  
+  .content-wrapper {
+    width: 80%;
+    padding-left: 20px;  
+    border-left: 1px solid rgba(0, 0, 0, 0.15);
+  }
+  
+  .content-wrapper, .sidebar-wrapper {
+    float: left;
+    padding-top: 20px;  
+  }
+`;
 
```

Comment:
Per discussion with @andrewbiang888 -- going to refactor the css in a separate PR